steps/http/session.go

```diff
@@ -74,11 +80,10 @@ func (s *Session) URL() (*url.URL, error) {
 	if err != nil {
 		return nil, fmt.Errorf("invalid endpoint URL '%s': %w", s.Request.Endpoint, err)
 	}
-	u, err = u.Parse(s.Request.Path)
-	if err != nil {
-		return nil, fmt.Errorf("invalid path URL '%s': %w", s.Request.Path, err)
+	u.Path = path.Join(u.Path, s.Request.Path)
```

Comment:
Considering this guideline: https://hackernoon.com/never-forget-anything-before-after-and-while-coding-98d187ae4cf1 and the fact that is not intuitive that Join removes the last slash, I would suggest to add a note as a reminder for the whole change (maybe HACK or NOTE)